author/topics/document-format/sections.adoc

```diff
@@ -40,7 +40,7 @@ that provides the English canonical form of the section name. For example:
 == Obligations
 
 The obligation of a section—whether it is normative or informative—is indicated
-with the attribute "obligation".
+with the attribute `obligation=`.
```

Comment:
Unless there’re objections, since it’s not part of attribute name per se I’m probably going to strip the trailing equality sign in such cases (but will point the reader to examples, so that they see how the value is passed to the attribute).

Reply:
OK